api/main.py

```diff
@@ -127,6 +127,12 @@ async def authorize_user(node_id: str, user: User = Depends(get_current_user)):
     # Only the user that created the node or any other user from the permitted
     # user groups will be allowed to update the node
     node_from_id = await db.find_by_id(Node, node_id)
+    # if node_from_id is NoneType - return 404
+    if not node_from_id:
+        raise HTTPException(
+            status_code=status.HTTP_404_NOT_FOUND,
+            detail=f"Node not found"
```

Comment:
Just to make the error message verbose.
```suggestion
            detail=f"Node not found with id: {node_id}"
```

Reply:
I don't think that's an issue here as the user submitting node data is already aware of the information sent to API.